ci/dockerfiles/ansible-e2e-hybrid.Dockerfile

```diff
@@ -23,16 +23,21 @@ ENV OPERATOR=/usr/local/bin/ansible-operator \
 # Ensure fresh metadata rather than cached metadata in the base by running
 # yum clean all && rm -rf /var/yum/cache/* first
 RUN yum clean all && rm -rf /var/cache/yum/* \
- && (yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm || true) \
- && yum install -y python-devel gcc inotify-tools \
- && easy_install pip \
- && pip install -U --no-cache-dir setuptools pip \
- && pip install --no-cache-dir --ignore-installed ipaddress \
-      ansible-runner==1.2 \
-      ansible-runner-http==1.0.0 \
-      openshift==0.8.9 \
-      ansible==2.8 \
- && yum remove -y gcc python-devel \
+ && (yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm || true) \
+ && yum install -y python36-devel.x86_64 gcc \
+ # Install inotify-tools
+ && yum install -y epel-release \
+ && (yum update || true) \
+ && curl -O https://rpmfind.net/linux/fedora/linux/releases/30/Everything/x86_64/os/Packages/i/inotify-tools-3.14-16.fc30.x86_64.rpm \
+ && rpm -i inotify-tools-3.14-16.fc30.x86_64.rpm \
+ && yum --enablerepo=epel install inotify-tools.x86_64 \
+ && (yum install python3-setuptools || true) \
+ && pip3 install --no-cache-dir --ignore-installed ipaddress \
+           ansible-runner==1.3.4 \
```

Comment:
Hi @joelanford, 

Besides, be possible to test it locally, it takes a lot to be checked. I did many tests in order to solve the issue "Error: failed to exec []string{“molecule”, “test”, “-s”, “test-local”}: exit status 1 Failed on the action: converge) See here <https://travis-ci.org/operator-framework/operator-sdk/jobs/584050528#L2442". 
 
The issue is related to the python versions and molecule since I could check all working ( master branch with just the download of the inotify-tools. In this way, how we are upgrading the versions shows that is ok and good indeed update the ansible-runner from 1.2 to 1.3.4. Also, it should be following the semversion which means that no break changes should be faced with.  

If we check the [Changelog](https://github.com/ansible/ansible-runner/blob/master/CHANGES.rst) I have the impression that these fixes are related to the issue faced before. 

@fabianvf I think you are the person who most have knowledge over it, have you any objection?  Have any reason for we keep using the old version?